### DISCUSSION TITLE
PlotSelectionOverTime in trame

### DISCUSSION THREAD

**Discussion by MarinsLucas** [Kitware/trame#430] [Trame/ParaView]
I'm trying to code the paraview tool "PlotSelectionOverTime" in my trame aplication. I already got how to plot the graph with plotly, but now I'm having difficulty extracting the Scalars values ​​from a cell over time. 
How can I do this? How do I access information from a cell in my mesh?

**Comment by jourdain** [comment]
You might be better serve on [ParaView Discourse](https://discourse.paraview.org/) but from the proxy `PlotSelectionOverTime` you can get its output dataset by doing the following.

```python
# won't work in distributed mode (use fetch instead)
vtk_dataset = plotSelectionOverTime.GetClientSideObject().GetOutput()
# ... extract arrays from vtk_dataset into numpy for plotly to use
```

**Comment by Rafaelrr5** [comment]
Hey @jourdain! I'm trying to get this kind of info that is given by the function, but I don't know if I should just put in there. I've this code here and do you know if I just put this ".GetClientSideObject().GetOutput()" for the output I'll have all the info I need to plot the graph? From that I say the action potential etc. 
![image](https://github.com/user-attachments/assets/d03530cb-96db-4afb-9ab7-e96cf77ae1f9)
The code is a little messed up because I've tried many things and I printed out the way it was already.

**Reply by jourdain** [reply]
Are you just missing the link between ‘output’ and ‘dataset’? Did you print the output to figure out what it is so you can make method calls on it to get the data you want? After that google will redirect you to the doc and api of such class from VTK..
